src/qt/utilitydialog.cpp

```diff
@@ -20,6 +20,9 @@
 #include "clientversion.h"
 #include "init.h"
 #include "util.h"
+#ifdef ENABLE_WALLET
```

Comment:
> possible solution would be to have a signal for collecting help strings

Certainly possible to have a signal for wallet to pass its help strings to server, but qt also has its own help strings (bitcoind should also have its own help string for `-server`, but that's actually implemented in init.cpp for now). Difficult to see where/how those would all be sorted.

Really I don't think it matters too much, and I'm happy with other approaches, or to drop this PR entirely if people don't think this is an improvement.